File: CPP/Clothing.cpp
#include <bits/stdc++.h>
#include "Product.cpp"

using namespace std;
class Clothing : public Product
{
    private:
    string size;
    string material;
    string gender;

    public:
    Clothing()
    {
        this->size = "";
        this->material = "";
        this->gender = "";
    }

    Clothing(string size, string material, string gender)
    {
        this->size = size;
        this->material = material;
        this->gender = gender;
    }

    string getSize()
    {
        return this->size;
    }

    void setSize(string size)
    {
        this->size = size;
    }

    string getMaterial()
    {
        return this->material;
    }

    void setMaterial(string material)
    {
        this->material = material;
    }

    string getGender()
    {
        return this->gender;
    }

    void setGender(string gender)
    {
        this->gender = gender;
    }

    ~Clothing()
    {
        
    }
};

File: python/Clothing.py
from Product import Product

class Clothing(Product):
    def __init__(self):
        super().__init__()
        self.size = ""
        self.material = ""
        self.gender = ""

    def __init__(self, size="", material="", gender=""):
        super().__init__()
        self.size = size
        self.material = material
        self.gender = gender

    def getSize(self):
        return self.size

    def setSize(self, size):
        self.size = size

    def getMaterial(self):
        return self.material

    def setMaterial(self, material):
        self.material = material

    def getGender(self):
        return self.gender

    def setGender(self, gender):
        self.gender = gender

    # def display(self):
        # return super().display() + " {:<4s} | {:<8s} | {:<6s} |".format(self.size, self.material, self.gender)
